pkgs/applications/misc/hugo/deps.nix

```diff
@@ -1,5 +1,23 @@
+# file generated from go.mod using vgo2nix (https://github.com/adisbladis/vgo2nix)
 [
-
+  {
+    goPackagePath = "github.com/gobuffalo/envy";
+    fetch = {
+      type = "git";
+      url = "https://github.com/gobuffalo/envy";
```

Comment:
I have not found any references to this in the go code itself.
Do you know where this is used?

Reply:
In case it is a test dependency only. The following will help:

```
subPackages = [ "." ];
```

this will make it build only the main package.

Reply:
Thanks, I’ll look into it tomorrow. It wouldn’t build without them though.